### DISCUSSION TITLE
[MacCatalyst] PointerEntered and PointerMoved not working

### DISCUSSION THREAD

**Discussion by TopperDEL** [unoplatform/uno#12425] [Q&A]
On MacCatalyst the Events `PointerEntered`, `PointerMoved` and `PointerExited` do not get raised. `PointerMoved` works if I click at a location.

Is this expected? what could I do to overcome this? And there does not seem to exist an open issue for that, right? Should I create one?

**Comment by jeromelaban** [comment]
Thanks for the report! Indeed, this is not supposed to happen, it's likely a specifity of iOS that's bleeding into Catalyst. If you can open an issue, that'll be great, thanks!

**Reply by TopperDEL** [reply]
Thanks @jeromelaban! I've create #12467.
